app/models/spotlight/resources/oaipmh_mods_parser.rb

```diff
@@ -179,14 +179,14 @@ def fetch_ids_uri(uri_str)
       end
     end
 
-    # Returns the uri for the iiif
-    def transform_ids_uri_to_iiif(ids_uri)
-      #Strip of parameters
+    # Returns the uri for the iiif thumbnail
+    def transform_to_iiif_thumbnail(ids_uri)
+      # Strip out parameters
       uri = ids_uri.sub(/\?.+/, '')
-      #Change /view/ to /iiif/
+      # Change /view/ to /iiif/
       uri = uri.sub(%r|/view/|, '/iiif/')
-      #Append /info.json to end
-      uri = uri.gsub('/full/150,/0/default.jpg', '')
+      # Append iiif format (thumbnail version)
+      uri = uri.gsub(/\/full\/\d*,\d*\/0\/default.jpg/, '')
```

Comment:
Reading notes: Instead of assuming the thumbnail of `150` this now allows for thumbnails of varied size.